tests/basic_crawl.test.js

```diff
@@ -8,9 +8,9 @@ test("ensure basic crawl run with docker run passes", async () => {
     'docker run -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler crawl --url https://example.com/ --generateWACZ  --text --collection wr-net --combineWARC --rolloverSize 10000 --workers 2 --title "test title" --description "test description" --warcPrefix custom-prefix',
   );
 
-  child_process.execSync(
-    "docker run -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler wacz validate --file collections/wr-net/wr-net.wacz",
-  );
+  // child_process.execSync(
```

Comment:
I wonder if it's worth using py-wacz just in some of the tests to validate that the resulting WACZs are valid, or implementing validation in the new `wacz` crawler module to do the same. They do pass validation in my manual testing but would be good to have that in the test suite to catch any potential regressions moving forward given how important this step of the crawling process is.

Reply:
yeah, we can run py-wacz outside of Docker and install just for CI, added that